website/src/components/MapContainer.tsx

```diff
@@ -0,0 +1,115 @@
+import { Extent } from '@arcgis/core/geometry';
+import VectorTileLayer from '@arcgis/core/layers/VectorTileLayer';
+import EsriMap from '@arcgis/core/Map';
+import MapView from '@arcgis/core/views/MapView';
+import LayerSelector from '@ugrc/layer-selector';
+import { useEffect, useRef, useState } from 'react';
+import useMap from './hooks/useMap';
+
+import '@ugrc/layer-selector/src/LayerSelector.css';
+
+const urls = {
+  landownership:
+    'https://gis.trustlands.utah.gov/hosting/rest/services/Hosted/Land_Ownership_WM_VectorTile/VectorTileServer',
+  liteVector:
+    'https://www.arcgis.com/sharing/rest/content/items/77202507796a4d5796b7d8e6871e352e/resources/styles/root.json',
+};
+
+type LayerFactory = {
+  Factory: new () => __esri.Layer;
+  url: string;
+  id: string;
+  opacity: number;
+};
+type SelectorOptions = {
+  view: MapView;
+  quadWord: string;
+  baseLayers: Array<string | { token: string; selected: boolean } | LayerFactory>;
+  overlays?: Array<string | LayerFactory>;
+  position: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+};
+
+const stateOfUtahExtent = {
+  xmax: -12612006,
+  xmin: -12246370,
+  ymax: 5125456,
+  ymin: 4473357,
+  spatialReference: {
+    wkid: 3857,
+  },
+};
```

Comment:
we have this [exported from kitchen sink](https://github.com/agrc/kitchen-sink/blob/main/packages/utilities/src/hooks/useDefaultExtent.ts#L4C1-L10C4) if you wanted to use that instead.